easybuild/easyconfigs/c/CRF++/CRF++-0.57-goalf-1.1.0-no-OFED.eb

```diff
@@ -12,8 +12,9 @@ description = """CRF++ is a simple, customizable, and open source implementation
 toolchain = {'name': 'goalf', 'version': '1.1.0-no-OFED'}
 toolchainopts = {'optarch': True, 'pic': True, 'opt': True}
 
+# manual download from https://taku910.github.io/crfpp/#download (via Google Drive)
```

Comment:
because it's not clear whether that corresponds to which version

the checksum included here makes very sure we're getting the right source tarball